Fix VERSION.
Use gnutls instead of openssl.
Fix overlinking.
Add -L flag for MacPorts libraries.
Fix mandir.
--- Makefile.orig	2021-02-19 11:54:54.000000000 -0600
+++ Makefile	2022-04-22 22:41:08.000000000 -0500
@@ -1,4 +1,4 @@
-VERSION=v2.4
+VERSION=2.4
 
 prefix=/usr/local
 
@@ -8,13 +8,13 @@
 SYS=posix
 #SYS=mingw
 
-CRYPTO=OPENSSL
+#CRYPTO=OPENSSL
 #CRYPTO=POLARSSL
-#CRYPTO=GNUTLS
+CRYPTO=GNUTLS
 LIBZ=-lz
-LIB_GNUTLS=-lgnutls -lhogweed -lnettle -lgmp $(LIBZ)
-LIB_OPENSSL=-lssl -lcrypto $(LIBZ)
-LIB_POLARSSL=-lpolarssl $(LIBZ)
+LIB_GNUTLS=-L$(prefix)/lib -lgnutls $(LIBZ)
+LIB_OPENSSL=-L$(prefix)/lib -lssl -lcrypto $(LIBZ)
+LIB_POLARSSL=-L$(prefix)/lib -lpolarssl $(LIBZ)
 CRYPTO_LIB=$(LIB_$(CRYPTO))
 DEF_=-DNO_CRYPTO
 CRYPTO_DEF=$(DEF_$(CRYPTO))
@@ -26,7 +26,7 @@
 
 bindir=$(prefix)/bin
 sbindir=$(prefix)/sbin
-mandir=$(prefix)/man
+mandir=$(prefix)/share/man
 
 BINDIR=$(DESTDIR)$(bindir)
 SBINDIR=$(DESTDIR)$(sbindir)
